src/main/java/lotto/step2/domain/LotteryCompany.java

```diff
@@ -0,0 +1,73 @@
+package lotto.step2.domain;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+import java.util.*;
+
+public class LotteryCompany {
+
+    private static final String COMMA = ",";
+    private static final String RATE_PATTERN = "0.##";
+    private final Map<Integer, Integer> prizeMoney = new HashMap<>();
+    private final List<Integer> winningNumbers = new ArrayList<>();
+    private Lotteries lotteries;
+
+    public LotteryCompany(String winningNumbers) {
+        setPrizeMoney();
+        setWinningNumbers(winningNumbers);
+    }
+
+    public LotteryCompany(String winningNumbers, Lotteries lotteries) {
+        this(winningNumbers);
+        this.lotteries = lotteries;
+    }
+
+    public Map<Integer, Integer> getPrizeMoney() {
+        return prizeMoney;
+    }
+
+    public List<Integer> getWinningNumbers() {
+        return winningNumbers;
+    }
+
+    public int getTotalCountOfMatch(int expected) {
+        return (int) lotteries.getLotteries().stream()
+                .filter(lotto -> lotto.isMatchExpected(winningNumbers, expected))
+                .count();
+    }
+
+    public double getRateOfReturn(int orderPrice) {
+        DecimalFormat decimalFormat = new DecimalFormat(RATE_PATTERN);
+        decimalFormat.setRoundingMode(RoundingMode.DOWN);
+        return Double.parseDouble(decimalFormat.format(calculateRateOfReturn() / orderPrice));
+    }
+
+    private void setPrizeMoney() {
+        prizeMoney.put(3, 5000);
```

Comment:
```suggestion
        prizeMoney.put(3, 5_000);
```
위와 같이 언더바를 통해 가독성을 높일 수 있습니다ㅎㅎ